src/aurelia-loader-webpack.ts

```diff
@@ -1,52 +1,9 @@
-/// <reference path="webpack-module.d.ts" />
```

Comment:
I'm not sure TypeScript will pick up the `d.ts` when it's a module, without the explicit reference to the file. I might be wrong here though.